File: backend/src/main/java/br/ufpr/sept/so2/shared/domain/exception/CredenciaisInvalidasException.java
package br.ufpr.sept.so2.shared.domain.exception;

public class CredenciaisInvalidasException extends RuntimeException {

    public static final String MENSAGEM =
            "Credenciais inválidas. Verifique seus dados e tente novamente.";

    public CredenciaisInvalidasException() {
        super(MENSAGEM);
    }
}

File: backend/src/main/kotlin/br/ufpr/sept/so2/shared/domain/exception/CredenciaisInvalidasException.kt
package br.ufpr.sept.so2.shared.domain.exception

class CredenciaisInvalidasException : RuntimeException(MENSAGEM) {
    companion object {
        const val MENSAGEM: String =
            "Credenciais inválidas. Verifique seus dados e tente novamente."
    }
}
